### DISCUSSION TITLE
How to Create Widgets and Shortcut cards in Zepp OS App Development？

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#115] [Blog]
With the recently fast-growing Zepp OS global developer ecosystem, more people are asking to learn how to create widgets in the Zepp OS app Development environment.

If you are new to Zepp OS app development, we would love to have you in our Eventbrite and Slack community and help you learn more about the potential power of Zepp OS.  Please help us spread the invitation to all!

Learn How to Build Primary and Secondary Widgets with Zepp OS App Development APIs and Tools.

https://www.eventbrite.com/e/how-to-create-primary-secondary-widgets-in-the-zepp-os-app-development-tickets-538242968207

**Comment by Xuemuyang** [comment]
Thanks to all the concerned developers, the event is successfully concluded.

The relevant sections of the documentation platform have also been rewritten and updated https://docs.zepp.com/docs/guides/framework/device/secondary-widget/
